testdata/baselines/reference/submoduleAccepted/compiler/thisAssignmentInNamespaceDeclaration1.errors.txt.diff

```diff
@@ -1,27 +1,19 @@
 --- old.thisAssignmentInNamespaceDeclaration1.errors.txt
 +++ new.thisAssignmentInNamespaceDeclaration1.errors.txt
 @@= skipped -0, +0 lines =@@
--a.js(1,8): error TS8006: 'module' declarations can only be used in TypeScript files.
+ a.js(1,8): error TS8006: 'module' declarations can only be used in TypeScript files.
  a.js(2,5): error TS2331: 'this' cannot be referenced in a module or namespace body.
 -b.js(1,11): error TS8006: 'namespace' declarations can only be used in TypeScript files.
++b.js(1,11): error TS8006: 'module' declarations can only be used in TypeScript files.
  b.js(2,5): error TS2331: 'this' cannot be referenced in a module or namespace body.
 
 
--==== a.js (2 errors) ====
-+==== a.js (1 errors) ====
-     module foo {
--           ~~~
--!!! error TS8006: 'module' declarations can only be used in TypeScript files.
-         this.bar = 4;
-         ~~~~
- !!! error TS2331: 'this' cannot be referenced in a module or namespace body.
-     }
-     
--==== b.js (2 errors) ====
-+==== b.js (1 errors) ====
+@@= skipped -15, +15 lines =@@
+ ==== b.js (2 errors) ====
      namespace blah {
--              ~~~~
+               ~~~~
 -!!! error TS8006: 'namespace' declarations can only be used in TypeScript files.
++!!! error TS8006: 'module' declarations can only be used in TypeScript files.
```

Comment:
This one appears to be using the wrong name; it should mention "namespace", not "module".